java/src/jmri/jmrit/display/layoutEditor/TrackSegment.java

```diff
@@ -1,15 +1,14 @@
 package jmri.jmrit.display.layoutEditor;
 
-import static jmri.jmrit.display.layoutEditor.LayoutTrack.NONE;
 import static jmri.util.MathUtil.add;
```

Comment:
`-import static jmri.jmrit.display.layoutEditor.LayoutTrack.NONE;
`
In this case I should always use `LayoutTrack.NONE` in my code (and remove the `-import static`. 
For all the `MathUtil` functions… I'd think their use obvious (being a math-head) and not too difficult to find… and the non-`-import static` code is too verbose (little value added).
But again… YOMV… ;-)